benchexec/tools/verioover.py

```diff
@@ -0,0 +1,47 @@
+
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    VeriOover
```

Comment:
Please add some more information about the tool here, e.g., at least a link to its homepage.